src/AM.Condo.IO/Diagnostics/ProcessInvoker.cs

```diff
@@ -19,6 +19,7 @@ public class ProcessInvoker : IProcessInvoker
     {
         #region Fields
         private readonly ILogger logger;
+        private readonly bool isRealtime = false;
```

Comment:
Is this necessary? Does it negatively impact us to have logging here all the time? If the logs are too verbose; perhaps we make the verbosity configurable, but I'm not a fan of disjointed flags like this. What does it mean to be realtime? how does the behavior change of this... thing?

:)

Reply:
Good point. originally everything was added to a queue and then output after execution. This made no sense when running MSBuild since some projects can take a while to run and giving feedback is important. 

I have seen large output of text create a slowdown in execution before but for our use-case i doubt it would be an issue.

I like the idea of adding verbosity into the mix, but would need to work out some details